### PR TITLE
added HomePage

### PR DESCRIPTION
**Pull request by avvaland26**
is it ok?

### CODE REVIEW

**Comment by mital20** [comment]
all good

File: src/main/java/WebTest/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package WebTest;
+
+public class HomePage extends Util{
+}
```
Comment (mital20) [inline src/main/java/WebTest/HomePage.java:4]: add something here

**Review by mital20 — APPROVED** [review]

File: src/main/java/WebTest/HomePage.java (hunk 1 above)
Comment (jignasharma) [reply src/main/java/WebTest/HomePage.java:4]: ok thanks

**Review by jignasharma — APPROVED** [review]

**Comment by avvaland26** [comment]
ok

File: src/main/java/WebTest/HomePage.java (hunk 1 above)
Comment (avvaland26) [reply src/main/java/WebTest/HomePage.java:4]: looks ok

Comment (mital20) [reply src/main/java/WebTest/HomePage.java:4]: ok

**Comment by jignasharma** [comment]
why

**Pull request merged**